Explorer/Assets/DCL/Multiplayer/Profiles/Tables/EntityParticipantTable.cs

```diff
@@ -12,8 +13,11 @@ public class EntityParticipantTable : IEntityParticipantTable
         private readonly Dictionary<string, Entity> walletIdToEntity = new ();
         private readonly Dictionary<Entity, string> entityToWalletId = new ();
 
-        public Entity Entity(string walletId) =>
-            walletIdToEntity[walletId].EnsureNotNull();
+        public Entity Entity(string walletId)
+        {
+            try { return walletIdToEntity[walletId].EnsureNotNull(); }
```

Comment:
You actually should add `where T : class` to `EnsureNotNull`. E.g. here it's redundant: `Entity` is a structure.